vaxrank/report.py

```diff
@@ -115,7 +122,7 @@ def _variant_data(self, top_vaccine_peptide):
         """
         variant_data = OrderedDict()
         mutant_protein_fragment = top_vaccine_peptide.mutant_protein_fragment
-        top_score = round(top_vaccine_peptide.combined_score, 4)
+        top_score = round(top_vaccine_peptide.combined_score, 7)
```

Comment:
A little bummed that you have to repeat `7` everywhere. 

At least, factor out the constant. 

Maybe show `1e-6` for small numbers?